jupyterlab_gcedetails/src/components/chart_wrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { AreaChart, Area, Tooltip, CartesianGrid, YAxis } from 'recharts';
+
+type ChartProps = {
```

Comment:
By convention we've used the `interface` keyword as opposed to `type` for defining object types. Is there a reason for choosing the latter?

Reply:
No particular reason, will update